backend/src/definitions/index.ts

```diff
@@ -2,3 +2,5 @@ export * from "./database";
 export * from "./table";
 export * from "./user-query";
 export * from "./postgres-adapter";
+export * from "./relation";
+export * from "./pipeline";
```

Comment:
We likely want to change the name pipeline given this new context. What do you think? Is there  clearly better name or is this good enough?

Reply:
Some options:
- JsonQuery
- StepQuery
- PreQuery
- PreSQL
- ProtoSQL
- ProtoQuery